code/modules/cargo/packs/science.dm

```diff
@@ -88,10 +164,30 @@
 	crate_name = "shield generators crate"
 	crate_type = /obj/structure/closet/crate/secure/science
 
+/datum/supply_pack/science/slime
+	name = "Slime Core Crate"
+	desc = "Ran out of slimes? No problem, contains one gray slime core. Requires Xenobio access to open"
```

Comment:
```suggestion
	desc = "Ran out of slimes? No problem, contains one gray slime core. Requires Xenobio access to open.”
```
Wouldn’t be too bothered allowing this to be not locked.

Reply:
traitor goal is to steal xenobio slime core so thats why this is locked. Ment to be more so for that kinda thing.